aaudio/README.md

```diff
@@ -1,15 +1,40 @@
 AAudio Samples
 ==============
 These samples demonstrate AAudio API:
-1. hello-aaudio sample: creates output stream
-1. echo sample: creates output and input streams, directly play captured audio
+1. hello-aaudio sample: creates an output stream
+1. echo sample: creates output && input streams, then loopback audio
 
-Requirements
+Pre-requisites
 -----------
-* Android Device with Android API 25 ( Andorid O )
-* A headphone( or a speaker ) plugged into Android Device
-* Support Audio Recording
-* Android Studio 2.3.0+
+* Android Device with recording capability, running Android API 25( Andorid O )
+* A external headphone( with a microphone ) plugged into Android Device
+* [Android Studio 2.3.0+](https://developer.android.com/studio/index.html)
+
+Getting Started
+---------------
+1. [Download Android Studio](http://developer.android.com/sdk/index.html)
+1. Launch Android Studio
+1. Import project: File --> New --> Import Project; browse to aaudio/build.gradle; and "OK"
+1. Open *File/Project Structure...*
+  - Click *Download* or *Select NDK location*.
+1. Click *Tools/Android/Sync Project with Gradle Files*.
+1. Click *Run/Run 'app'*.
+
+Screenshots
+-----------
+![screenshot](screenshot.png)
+
+Support
+-------
+If you've found an error in these samples, please [file an issue](https://github.com/googlesamples/android-audio-high-performance/issues/new).
+
+Patches are encouraged, and may be submitted by [forking this project](https://github.com/googlesamples/android-audio-high-performance/fork) and
+submitting a pull request through GitHub. Please see [CONTRIBUTING.md](../CONTRIBUTING.md) for more details.
+
+- [Stack Overflow](http://stackoverflow.com/questions/tagged/android-ndk)
```

Comment:
no, probablly pickback to NDK for now.  As this repo grows, might be a good idea to add a new tag